gamification/src/Components/Navbar/Navbar.js

```diff
@@ -12,18 +12,13 @@ function Navbar() {
         <BNavbar.Collapse id="basic-navbar-nav">
           <Nav className="me-auto">
             <Nav.Link href="/courses">Cursos</Nav.Link>
-            <Nav.Link href="/teachergroups">Grupos</Nav.Link /*TODO why two groups for teacher*/> 
+            <Nav.Link href="/teachersubjects">Grupos</Nav.Link /*TODO why two groups for teacher*/> 
             <Nav.Link href="/periods">Periodos</Nav.Link>
             <Nav.Link href="/missions">Misiones</Nav.Link>
-            <NavDropdown title="Dropdown" id="basic-nav-dropdown" /*TODO add logout here*/>
-              <NavDropdown.Item href="#action/3.1">Action</NavDropdown.Item>
-              <NavDropdown.Item href="#action/3.2">
-                Another action
-              </NavDropdown.Item>
-              <NavDropdown.Item href="#action/3.3">Something</NavDropdown.Item>
+            <NavDropdown title="Más..." id="basic-nav-dropdown" /*TODO add logout here*/>
```

Comment:
make dynmac with circuit break
remove dropdown only have button

Reply:
Done. Only missing to add logout logic and align the button to rightmost edge. Will finish with all assigned change requests first though.

Reply:
Remove dropdown done. Only missing to add logout logic and align the button to rightmost edge. Will finish with all assigned change requests first though.